### ISSUE TITLE
slices: package slices is not in GOROOT

### ISSUE THREAD

**Issue by lei0320** [onsi/gomega#811]
run & got:
```
$ go get github.com/onsi/gomega/...
github.com/onsi/gomega/matchers/support/goraph/bipartitegraph imports
        slices: package slices is not in GOROOT (/Users/xxx/go/go1.18/src/slices)
```

go env:
```
GO111MODULE="on"
GOARCH="arm64"
GOBIN=""
GOCACHE="/Users/xxx/Library/Caches/go-build"
GOENV="/Users/xxx/Library/Application Support/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=" -mod=readonly"
GOHOSTARCH="arm64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/xxx/go/pkg/mod"
GONOPROXY="*.xxx.com"
GONOSUMDB="*.xxx.com"
GOOS="darwin"
GOPATH="/Users/xxx/go"
GOPRIVATE="*.xxx.com"
GOPROXY="xxx,direct"
GOROOT="/Users/tal/go/go1.18"
GOSUMDB="off"
GOTMPDIR=""
GOTOOLDIR="/Users/xxx/go/go1.18/pkg/tool/darwin_arm64"
GOVCS=""
GOVERSION="go1.18"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/xxx/xxx/xx/xx/go.mod"
```

os info：
```
Apple M3
mac OS 14.5
```

**Comment by lei0320** [comment]
and when i run a test, it output:
```
../../../go/pkg/mod/github.com/onsi/gomega@v1.36.2/matchers/support/goraph/bipartitegraph/bipartitegraphmatching.go:4:2: package slices is not in GOROOT (/Users/xxx/go/go1.18/src/slices)
```

**Comment by thediveo** [comment]
Hi there! It seems you are using a Go version 1.18 which is unsupported by the current Gomega (and Ginkgo) versions. We follow the same support strategy of the Go toolchain itself, supporting the current stable as well as the previous stable. As there have been several CVEs on older Go versions that are fixed only in more recent versions, we recommend upgrading your Go toolchain.

Alternatively, you need to get a much older version of Gomega that was released in the Go 1.18-1.19 time frame. However, on macos there is the danger that newer Go version sometimes drop support for older macos version.

(As there are many free Linux distributions this tends to not be a problem on Linux where you can most of the time update hardware that has long since declared obsolete by its makers, in contrast to Apple.)
